modulefiles/ufs_linux.intel

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
This file has been missing ever since, even though the `cmake` include file referenced in line 19 exists. Adding it here so that linux/macos gnu/intel are complete.